src/applications/personalization/profile/components/direct-deposit/DirectDepositV1.jsx

```diff
@@ -26,19 +36,36 @@ const DirectDeposit = () => {
       >
         Direct deposit information
       </h2>
-      <DowntimeNotification
-        appTitle="direct deposit"
-        render={handleDowntimeForSection(
-          'direct deposit for compensation and pension',
-        )}
-        dependencies={[externalServices.evss]}
-      >
-        <BankInfoCNP />
-      </DowntimeNotification>
+      {showSetUp2FactorAuthentication && (
+        <SetUp2FAAlert isAuthenticatedWithSSOe={isAuthenticatedWithSSOe} />
+      )}
+      {!showSetUp2FactorAuthentication && (
```

Comment:
Am I understanding correctly that (aside from the double 2FA alert issue)

- previously, we were rendering the downtime notification above the 2FA alert, the downtime notification seemed independently rendered unrelated to the 2FA status
- now we can only render either the downtime notification OR the 2FA alert?

I guess my question is.. do we need to wrap `<BankInfoCNP />` with the `!showSetUp2FactorAuthentication` conditional instead of `DowntimeNotification`?

Reply:
Previously, if there was a known downtime issue with EVSS, the `BankInfoCNP` would not render at all so you wouldn't even see the "set up 2FA" alert.

It looks like this change is blocking the `DowntimeNotification` component based on `showSetUp2FactorAuthentication` (which is technically is) but what it's really trying to do is not show the `BankInfoCNP` at all if the user does not have 2FA set up.